src/tools/cephfs_mirror/PeerReplayer.h

```diff
@@ -45,6 +45,30 @@ class PeerReplayer {
   inline static const std::string SERVICE_DAEMON_FAILED_DIR_COUNT_KEY = "failure_count";
   inline static const std::string SERVICE_DAEMON_RECOVERED_DIR_COUNT_KEY = "recovery_count";
 
+  typedef std::pair<std::string, uint64_t> Snapshot;
+
```

Comment:
nit: you might want to use the `using` directive here instead of `typedef`

Reply:
done